### PR TITLE
Swap to Debian slim baseimage

### LINKED ISSUE #402: Alternatives to alpine-glibc?
The Alpine Technical Steering Committee recommends against using the glibc package because of the potential for undefined behavior when loading system libraries (which, in Alpine, are linked against musl):

https://gitlab.alpinelinux.org/alpine/tsc/-/issues/17

Would the maintainers consider switching away from alpine-glibc? If so, would Debian or Debian-slim be a suitable base image? I'd be happy to submit a PR with the necessary changes.

### PR DESCRIPTION
**Pull request by Doridian**
Resolves #402

### CODE REVIEW

File: docker/Dockerfile [hunk 1]
```diff
@@ -35,7 +35,9 @@ RUN set -ox pipefail \
     fi \
     && archive="/tmp/factorio_headless_x64_$VERSION.tar.xz" \
     && mkdir -p /opt /factorio \
-    && apk add --update --no-cache --no-progress bash binutils curl file gettext jq libintl pwgen shadow su-exec \
+    && apt-get -y update \
+    && apt-get -y install binutils curl file gettext jq pwgen xz-utils \
+    && rm -rf /var/cache/apt \
```
Comment (SuperSandro2000) [inline docker/Dockerfile:40]:
```suggestion
    && rm -rf /var/lib/apt/lists/* \
```

File: docker/Dockerfile [hunk 2]
```diff
@@ -35,7 +35,9 @@ RUN set -ox pipefail \
     fi \
     && archive="/tmp/factorio_headless_x64_$VERSION.tar.xz" \
     && mkdir -p /opt /factorio \
-    && apk add --update --no-cache --no-progress bash binutils curl file gettext jq libintl pwgen shadow su-exec \
+    && apt-get -y update \
+    && apt-get -y install binutils curl file gettext jq pwgen xz-utils \
```
Comment (SuperSandro2000) [inline docker/Dockerfile:39]:
```suggestion
    && apt-get -q update \
    && apt-get -qy install binutils curl file gettext jq pwgen xz-utils \
```

**Comment by Doridian** [comment]
Closing this PR as it seems no PR on this repo is getting looked at anyway. Will continue maintaining my own fork for features I need.

**Comment by Doridian** [comment]
Sorry about that earlier comment. I basically realized I started running ahead updating the mod updater because `factorio_version` does not always work, you need the `/full` and parse out the `base` constraints.
Literally after I get actual comments on the PR going finally, which I completely forgot about at that time.

Anyway, if there is interest in either the swap to a slim baseimage, or a mod updater that checks full constraints or both, I am happy to open seperate PRs again. Just let me know.

**Pull request closed without merging**